rmf_demo_maps/maps/office/log.css

```diff
@@ -0,0 +1,53 @@
+table, th, td {
```

Comment:
What is this file for? Can it be removed?

Reply:
This file is automatically generated and shall be ignored.